service/iot-service-client/pom.xml

```diff
@@ -80,6 +80,16 @@
             <artifactId>slf4j-api</artifactId>
             <version>1.7.25</version>
         </dependency>
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-core</artifactId>
+            <version>1.12.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-core-amqp</artifactId>
```

Comment:
This dependency gives us a nice enum for dealing with Cbs authorization token types